.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Edgar Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.defaultInterpreterPath": "/usr/bin/python3.12"
+            },
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "ms-python.debugpy",
+                "ms-azuretools.vscode-docker",
+                "mechatroner.rainbow-csv"
+            ]
+        }
+    },
+    "postCreateCommand": "poetry install",
+    "remoteUser": "edgar"
```

Comment:
This breaks container creation, resulting in:

`unable to find user edgar: no matching entries in passwd file`

Because the image is a vscode devcontainer image, the vscode user is a pre-defined one I guess!

Reply:
Haha I was foolish for thinking it would be so easy!

I fixed this in https://github.com/bellingcat/EDGAR/pull/37/commits/2a261e136b6a65599a68a3fbb40bd42e36eb84a5 by removing the `remoteUser` key completely. Turns out it's optional and defaults to the container's default user - which in this case is `vscode`. So it's working again and maintains the same behavior, while keeping us safe from breaking changes if the container's base image ever changes the default user.